.github/workflows/codeql-analysis.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
     # Initializes the CodeQL tools for scanning.
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@v2
+      uses: github/codeql-action/init@294a9d92911152fe08befb9ec03e240add280cb3 # v3.26.8
```

Comment:
Breaking change in v2 -> v3 is node 20 support, see top of their changelog here: https://github.com/github/codeql-action/blob/main/CHANGELOG.md#codeql-action-changelog